src/components/nodes/pool/pool.vue

```diff
@@ -250,6 +251,7 @@ export default {
 
         this.getElementsUnderArea(element, this.graph).filter(laneElement => {
           return laneElement.component &&
```

Comment:
This expression is getting quite long and complicated. Could it be split into multiple calls to `filter()`? Another option would be to move it to a method and give it a descriptive name.

Reply:
I was going to move this whole block into another function as it's using an early return but it's hard to tell. I agree that this is a huge function that needs to be broken up though.

Reply:
Multiple calls to `.filter()` would make this more readable IMO.